exercises/rna-transcription/canonical-data.json

```diff
@@ -15,49 +15,49 @@
   ],
   "cases": [
     {
-      "description": "rna complement of cytosine is guanine",
+      "description": "RNA complement of cytosine is guanine",
       "property": "toRna",
       "dna": "C",
       "expected": "G"
     },
     {
-      "description": "rna complement of guanine is cytosine",
+      "description": "RNA complement of guanine is cytosine",
       "property": "toRna",
       "dna": "G",
       "expected": "C"
     },
     {
-      "description": "rna complement of thymine is adenine",
+      "description": "RNA complement of thymine is adenine",
       "property": "toRna",
       "dna": "T",
       "expected": "A"
     },
     {
-      "description": "rna complement of adenine is uracil",
+      "description": "RNA complement of adenine is uracil",
       "property": "toRna",
       "dna": "A",
       "expected": "U"
     },
     {
-      "description": "rna complement",
+      "description": "RNA complement",
       "property": "toRna",
       "dna": "ACGTGGTCTTAA",
       "expected": "UGCACCAGAAUU"
     },
     {
-      "description": "dna correctly handles invalid input",
+      "description": "DNA correctly handles invalid input",
```

Comment:
had to double-take here. DNA is correct since the input is being transformed from DNA to RNA.